src/domain/swaps/entities/schemas/order.schema.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { orderBuilder } from '@/domain/swaps/entities/__tests__/order.builder';
+import { OrderSchema } from '@/domain/swaps/entities/schemas/order.schema';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+import { Order } from '@/domain/swaps/entities/order.entity';
+
+describe('OrderSchema', () => {
+  it('should validate a valid order', () => {
+    const order = orderBuilder().build();
+
+    const result = OrderSchema.safeParse(order);
+
+    expect(result.success).toBe(true);
+  });
+
+  it('hexadecimal signature should be valid', () => {
+    const order = {
+      ...orderBuilder().build(),
+      signature: faker.string.hexadecimal() as `0x${string}`,
```

Comment:
Is there a reason to now use `with` in these tests?

Reply:
No reason! Will fix it! 👍

Reply:
I've fixed it in [107cf08](https://github.com/safe-global/safe-client-gateway/pull/1302/commits/107cf0878fbb2dead19a4885feadb58ec472fd1c).

Additionally I merged the "unknown" tests into one (parameterised test).